x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/multi_page_layout/components/bottom_bar.tsx

```diff
@@ -125,7 +125,9 @@ export const AgentStandaloneBottomBar: React.FC<{
 export const CreatePackagePolicyFinalBottomBar: React.FC<{
   pkgkey: string;
 }> = ({ pkgkey }) => {
+  const isK8s = pkgkey.includes('kubernetes');
```

Comment:
nit we have a constant `FLEET_KUBERNETES_PACKAGE` that could be used here

Reply:
I will use that